src/FormField/AutoComplete.php

```diff
@@ -128,10 +128,15 @@ public function getData()
             }
         }
 
-        $data = $this->model->export([$this->model->id_field, $this->model->title_field]);
-
+        //fix #327: https://github.com/atk4/ui/issues/327
+        $data = [];
+        $res = $this->model->export([$this->model->id_field, $this->model->title_field]);
+        foreach ($res as $item) {
```

Comment:
This could be slow.
Simple tweak would be to wrap foreach in
`if ($this->model->id_field != 'id' || $this->model->title_field != 'name')`

Reply:
agreed, would deal with majority of cases.

Reply:
The wrap makes sense, though the performance issue shouldn't be massive as long as a sensible LIMIT (standard 10) is set.

Reply:
well ... if limit is always set and it's such small (10, 20 rows), then wrapping will not make performance better. So I accept these changes and let Romans decide to merge or not to merge :)